tests/test_simplearray.py

```diff
@@ -0,0 +1,20 @@
+from modmesh import SimpleArrayUint64, SimpleArrayFloat64
```

Comment:
I will prefer to put the code at [`test_buffer.py`](https://github.com/solvcon/modmesh/blob/master/tests/test_buffer.py`), where we put all other array tests.

Reply:
I concur.  Do not add a new test file for this simple test.